tools/cibadmin.c

```diff
@@ -73,6 +72,8 @@ void cibadmin_op_callback(xmlNode *msg, int call_id, int rc, xmlNode *output,
 static void
 print_xml_output(xmlNode * xml)
 {
+    char *buffer;
+
```

Comment:
we could have this live just in the block where it is needed

Reply:
Yeah, I was just reverting the code back to the way it was but there's no reason to not also make it better.  Updated.